index.d.ts

```diff
@@ -0,0 +1,49 @@
+declare type Acl = 'private' | 'bucketOwnerRead' | 'bucketOwnerFullControl' | 'projectPrivate' | 'authenticatedRead' | 'publicRead' | 'publicReadWrite';
+
+declare interface MulterOptions {
+    bucket: string;
+    projectId: string;
+    keyFilename?: string;
+    filename?: string;
+    destination?: string;
+    acl?: Acl;
+    resize?: boolean;
+    crop?: boolean;
+    background?: boolean;
+    embed?: boolean;
+    max?: boolean;
+    min?: boolean;
+    withoutEnlargement?: boolean;
+    ignoreAspectRatio?: boolean;
+    extract?: boolean;
+    trim?: boolean;
+    flatten?: boolean;
+    extend?: boolean;
+    negate?: boolean;
+    rotate?: boolean;
+    flip?: boolean;
+    flop?: boolean;
+    blur?: boolean;
+    sharpen?: boolean;
+    gamma?: boolean;
+    grayscale?: boolean;
+    greyscale?: boolean;
+    normalize?: boolean;
+    normalise?: boolean;
+    withMetadata?: boolean;
+    convolve?: boolean;
+    threshold?: boolean;
+    toColourspace?: boolean;
+    toColorspace?: boolean;
+    toFormat?: boolean;
+    gzip?: boolean;
+    metadata?: { [key: string]: any };
+}
+
+declare class MulterSharp {
+    constructor(options: MulterOptions)
+}
+
+declare function multerSharp(options: MulterOptions): MulterSharp;
+
+export default multerSharp;
```

Comment:
Just curious, because I used `module.exports` to export namespace multerSharp. So why `export default` in this definition?

Reply:
Because I did it wrong :smile:. Thanks for pointing that out so I could deep dive into exports. It has now been updated to 
```typescript
export = multerSharp;
```